ls1intum/Artemis
# `Communication`: Enable reactions if only messaging is enabled

### Checklist
#### General


#### Server

### Motivation and Context


Reactions only work if communication is enabled

### Description

Reactions now also work, if only messaging is enabled.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Instructor
- 1 Course with only messaging enabled

1. Check that you can react to messages and replies as both users and they get updated for the other user
2. Enable communication, disable messaging
3. Check that you can react to posts and replies as both users and reactions get updated for the other user

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots


No UI changes
